tensorboard/plugins/projector/projector_api_test.py

```diff
@@ -21,37 +21,66 @@
 import os
 import shutil
 
+import six
 import tensorflow as tf
 
 from google.protobuf import text_format
 
 from tensorboard.plugins import projector
 from tensorboard.util import test_util
 
-tf.compat.v1.disable_v2_behavior()
-
+def create_dummy_config():
+  return projector.ProjectorConfig(
+      model_checkpoint_path = 'test',
+      embeddings = [
+          projector.EmbeddingInfo(
+              tensor_name = 'tensor1',
+              metadata_path = 'metadata1',
+          ),
+      ],
+  )
 
 class ProjectorApiTest(tf.test.TestCase):
 
-  def testVisualizeEmbeddings(self):
-    # Create a dummy configuration.
-    config = projector.ProjectorConfig()
-    config.model_checkpoint_path = 'test'
-    emb1 = config.embeddings.add()
-    emb1.tensor_name = 'tensor1'
-    emb1.metadata_path = 'metadata1'
+  def test_visualize_embeddings_with_logdir(self):
+    logdir = self.get_temp_dir()
+    self.addCleanup(shutil.rmtree, logdir)
```

Comment:
No need: `get_temp_dir()` is cleaned up automatically, and every `rm -r`
that we can remove from the codebase makes me sleep a little easier. :-)

Reply:
Did not know! Thanks!